dnf/base.py

```diff
@@ -2500,6 +2498,29 @@ def _report_already_installed(self, packages):
         for pkg in packages:
             _msg_installed(pkg)
 
+    def _raise_package_not_found_error(self, pkg_spec, forms, reponame):
+        all_query = self.sack.query(flags=hawkey.IGNORE_EXCLUDES)
+        subject = dnf.subject.Subject(pkg_spec)
+        solution = subject.get_best_solution(
+            self.sack, forms=forms, with_src=False, query=all_query)
+        if reponame is not None:
+            solution['query'].filterm(reponame=reponame)
+        if not solution['query']:
+            raise dnf.exceptions.PackageNotFoundError(_('No match for argument'), pkg_spec)
+        else:
+            with_modular_query = self.sack.query(flags=hawkey.IGNORE_MODULAR_EXCLUDES)
+            with_modular_query = solution['query'].intersection(with_modular_query)
+            with_regular_query = self.sack.query(flags=hawkey.IGNORE_REGULAR_EXCLUDES)
+            with_regular_query = solution['query'].intersection(with_regular_query)
```

Comment:
No, it is impossible because regular excludes are applied first then modular filtering take place. Therefore modular excludes have only packages that were not previously excluded by regular filtering.